RELEASE_NOTES.md

```diff
@@ -3,6 +3,7 @@
 ## v3.0.0
 ### New features
 - The Identity Provider now uses the `webid` scope as required for Solid-OIDC.
+- The `VoidLocker` could be used as development/testing purposes without any locking mechanism.
```

Comment:
```suggestion
- The `VoidLocker` can be used to disable locking for development/testing purposes. This can be enabled by changing the `/config/util/resource-locker/` import to `debug-void.json`
```